doc/classes/AudioEffectCompressor.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectCompressor" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Compressor audio effect to an Audio bus.
+		Reduces sounds that exceed a certain threshold level, smooths out the dynamics and increases the overall volume.
 	</brief_description>
 	<description>
+		Dynamic range compressor reduce the level of the sound when the amplitude goes over a certain threshold in Decibels. One of the main uses of a compressor is to increase the dynamic range while clipping the least possible (when sound goes over 0dB).
+		Compressor has may uses in the mix :
+		- Uses in the Master bus to compress the whole output (Although a Limiter is probably better)
+		- Uses in voice channels to ensure they sound as even as possible.
+		- Sidechained. Means it can reduce the sound level using another audio bus for threshold detection. This technique is very common in video game mixing to download the level of Music/SFX while voices are being heard.
```

Comment:
Sentence has no subject.  Try:
"- Sidechained, which can reduce the sound level..."

download -> downmix or reduce